ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-ref.yaml

```diff
@@ -38,14 +38,22 @@ ref:
     default: "false"
     documentation: |-
       Trigger to enable local indexing
+  - name: METRIC_PROFILES
```

Comment:
why introduce this? we already have `EXTRA_FLAGS`

Reply:
Good catch. Using extra flags itself.